docs/CHANGELOG.md

```diff
@@ -9,6 +9,124 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
+`ETH_DEFAULT_BATCH_SIZE` - EvmDefaultBatchSize controls the number of receipts fetched in each request in the EvmConfirmer
+`CLOBBER_NODES_FROM_ENV` - Defaulting to true, this env var when set will autocreate database rows for chain and nodes. It will upsert a new chain using the DefaultChainID and upsert nodes corresponding to the given ETH_URL and ETH_SECONDARY_URLS. It is recommended, after the initial population, to set this env var to false and thereafter to use the CLI commands or API to manage chains/nodes.
+
+#### Multichain support added
+
+Chainlink now supports connecting to multiple different EVM chains simultaneously.
+
+This means that one node can run jobs on Goerli, Kovan, BSC and Mainnet (for example).
+
+Extensive efforts have been made to make this as seamless as possible. Generally speaking, you should not have to make any changes when upgrading your existing node to this version. All your jobs will continue to run as before.
+
+The overall summary of changes is such:
+
+##### Chains/Ethereum Nodes
+
+EVM chains are now represented as a first class object within the chainlink node. You can create/delete/list them using the CLI or API, e.g.
+
+###### Creation
+
+```bash
+chainlink chains evm create -id  # creates an evm chain with chain ID 42 (see: https://chainlist.org/)
+chainlink nodes create -name 'my-primary-kovan-full-node' -type primary -ws-url ws://node.example/ws -http-url http://node.example/rpc # http-url is optional but recommended for primaries
+chainlink nodes create -name 'my-send-only-backup-kovan-node' -type sendonly -http-url http://some-public-node.example/rpc
+```
+
+Currently only a single primary node is supported, and one is required in order for a chain to connect. You may specify zero or more send-only nodes for a chain. In future, support for multiple primaries will be added.
+
+###### Listing
+
+```bash
+chainlink chains evm list
+chainlink nodes list
+```
+
+###### Deletion
+
+```bash
+chainlink nodes delete 'my-send-only-backup-kovan-node'
+chainlink chains evm delete 42
+```
+
+###### CLOBBER_NODES_FROM_ENV
+
+The old way of specifying chains using environment variables is still supported but discouraged. It works as follows:
+
+If you specify `CLOBBER_NODES_FROM_ENV` (default: true) then the values of `ETH_CHAIN_ID`, `ETH_URL`, `ETH_HTTP_URL` and `ETH_SECONDARY_URLS` will be used to create/update chains and nodes representing these values in the database. If an existing chain/node is found it will be overwritten. This environment variable is used mainly to ease the process of upgrading, and on subsequent runs (once your old settings have been written to the database) it is recommended to run with `CLOBBER_NODES_FROM_ENV=false` and use the API commands exclusively to administer chains and nodes.
```

Comment:
is it true that if any of the old envvars are set then we want to clobber the db? Presumably if someone moves to CLOBBER_NODES_FROM_ENV=false they could also just remove the old envvars. If so then we could infer the clobbering from the existence of the old envvars and avoid introducing CLOBBER_NODES_FROM_ENV

Reply:
Good point.

Reply:
I spoke to Peter about this and he wants it to be backwards compatible with the old way of using ENV vars to change node addresses etc. So I decided to leave this as-is.